documentation/mqtt-overview.md

```diff
@@ -0,0 +1,83 @@
+# AnkerMake MQTT Communications Overview
+
+We're going over the AnkerMake implementation of the [MQTT communications standard](https://mqtt.org/). While we won't be explaining the MQTT standard itself, we will explain certain aspects of how the protocol works for ease of understanding.
+
+## Generic MQTT Basics
+
+MQTT (Messaging Que Transport Telemetry) is a [stateful](https://byjus.com/gate/difference-between-stateless-and-stateful-protocol/#:~:text=What%20is%20a%20Stateful%20Protocol,Protocol%20(FTP)%2C%20etc.), [edge driven](https://en.wikipedia.org/wiki/Edge_computing), lightweight, and low bandwidth communications protocol primarily used by [IoT devices](https://en.wikipedia.org/wiki/Internet_of_things). It was specifically designed for industry, so that's the context in which we'll lay out our brief explanation.
+
+The two actors in this type of network are a **Broker** and a **Client**.  MQTT uses a publish/subscribe model to pass data and commands between clients facilitated by a broker. In order to organize where data and commands are supposed to go, the Broker organizes a hierarchy of **Topics**. 
+
+A Topic is simply a namespace that clients can subscribe and/or publish to. It can describe almost any level of abstraction in your network hierarchy, but always ends at a single client device parameter at the lowest level.
```

Comment:
"A Topic is simply a namespace that clients can subscribe and/or publish to."

That part is good. The last part of this paragraph is confusing.

Maybe we can compare it to participants in a chat system, such as IRC or discord? That should be a pretty close analogy.